### ISSUE TITLE
Better 404 page

### ISSUE THREAD

**Issue by colintate** [emmapersky/one-click-orgs#100]
http://rails3.oneclickorgs.com/members/40000 - should this be more cosmetically handled?  (still with messaging, but with easier nav ability to get back in?

**Comment by chrismear** [comment]
404 template needs to be added:

```
ActionView::MissingTemplate (Missing template //home/oneclickorgs/var/www/rails3.oneclickorgs.com/public/404.html with {:formats=>[:html]} in view path /home/oneclickorgs/var/www/rails3.oneclickorgs.com/app/views:/home/oneclickorgs/var/www/rails3.oneclickorgs.com/vendor/plugins/haml/app/views):
  app/controllers/application_controller.rb:95:in `render_404'
```

**Comment by chrismear** [comment]
The 404.html template is actually already there, but that ActionView::MissingTemplate is being caused by a Rails bug that's fixed in 3.0.0.beta2: http://github.com/rails/rails/commit/ae933a093db93c22b9facd3411afd9ef69719193

The template still needs prettying up, though.

**Comment by chrismear** [comment]
This ticket now at:

https://github.com/oneclickorgs/one-click-orgs/issues/100
